src/Common/tests/System/Diagnostics/RemoteExecutorConsoleApp/project.json

```diff
@@ -1,9 +1,6 @@
 {
   "dependencies": {
     "System.Runtime": "4.1.0-rc3-24011-00",
-    "Microsoft.DotNet.xunit.performance": "1.0.0-alpha-build0029",
```

Comment:
These weren't actually used in the test project, and were causing some build warnings due to "different version of the same dependent assembly", so I removed them from this project.